components/brave_wallet/browser/swap_request_helper.cc

```diff
@@ -307,6 +303,32 @@ std::optional<base::Value::Dict> EncodeStepEstimate(
   }
   result.Set("gasCosts", std::move(gas_costs_value));
 
+  if (!estimate->fee_costs) {
+    return result;
+  }
+
+  base::Value::List fee_costs_value;
+  for (const auto& fee_cost : *estimate->fee_costs) {
+    base::Value::Dict fee_cost_value;
+    fee_cost_value.Set("name", fee_cost->name);
+    fee_cost_value.Set("description", fee_cost->description);
+    fee_cost_value.Set("amount", fee_cost->amount);
+    fee_cost_value.Set("percentage", fee_cost->percentage);
+    fee_cost_value.Set("included", fee_cost->included);
+
+    // fake the USD amount value since it is not used by LiFi
+    fee_cost_value.Set("amountUSD", "0");
+
+    if (auto token = EncodeToken(std::move(fee_cost->token))) {
```

Comment:
fee_cost is const so std::move has no effect. Actually all calls to EncodeToken shold not recieve args by std::move